augur/merge.py

```diff
@@ -503,3 +579,85 @@ def quote(s):
     # …and instead quote a final empty string down here if we're still empty
     # after joining all our parts together.
     return quoted if quoted else shquote('')
+
+
+def run_command(command: str, print_stdout: bool):
+    """Run a command, clearly marking any messages that resemble SeqKit output."""
+    process = subprocess.Popen(
+        ['bash', '-euo', 'pipefail', '-c', command],
+        stdout=subprocess.PIPE if print_stdout else subprocess.DEVNULL,
+        stderr=subprocess.PIPE,
+        text=True,
+    )
+
+    if print_stdout:
+        for line in iter(process.stdout.readline, ''):
+            print(line.rstrip())
+
+    for line in iter(process.stderr.readline, ''):
+        # Remove ANSI escape sequences common in SeqKit output.
+        line = re.sub(r'\x1b\[[\d;]+m', '', line)
+
+        # Detect messages from seqkit and append a prefix.
+        if line.startswith("[INFO]"):
+            print_debug(f"[SeqKit] {line.rstrip()}")
+        elif line.startswith("[ERRO]"):
+            print_info(f"[SeqKit] {line.rstrip()}")
+        else:
+            print_info(line.rstrip())
```

Comment:
Why are we bothering with this? It makes the code more complicated, and I'm not sure of what we're gaining?

Reply:
Fair point. I thought it would be good to mark SeqKit's outputs more clearly. I now realize that there is already a [`run_shell_command`](https://github.com/nextstrain/augur/blob/e34a62bdc43a1e12cd8b58277f750a27bb2e20e7/augur/io/shell_command_runner.py#L14) helper function in this codebase, so I've made the switch to that which will mark SeqKit's outputs clearly when there is an error. It needs some tweaking to work nicely for the use cases in this PR, but I think it's better than the `run_command` function that I originally had here.